test/common/upstream/outlier_detection_impl_test.cc

```diff
@@ -1489,6 +1489,168 @@ TEST_F(OutlierDetectorImplTest, Consecutive_5xxAlreadyEjected) {
   loadRq(hosts_[0], 5, 500);
 }
 
+// Test verifies that ejection time increases each time the node is ejected,
+// and decreases when node stays healthy.
+#define EJECT_PERIOD_TICK(x) x * 10000
```

Comment:
Not sure what this adds, but in general prefer functions over macros for this kinda stuff

Reply:
It is just a helper. Instead of moving time using 10000, 20000, 30000, ..... 300000, this allows to use 1, 2,3, ... 30.
Converted it to constexpr function.